config/sections/mixins/headline.php

```diff
@@ -6,9 +6,16 @@
     'props' => [
         /**
          * The headline for the section. This can be a simple string or a template with additional info from the parent page.
+         * @todo deprecate in 3.8
          */
         'headline' => function ($headline = null) {
             return I18n::translate($headline, $headline);
+        },
+        /**
+         * label is the new official replacement for headline
+         */
+        'label' => function ($label = null) {
```

Comment:
One question (with no answer myself): if we talk about making things more consistent and fixing `name` with `title`... should it all be `title` eventually?

Reply:
👍